dapr/proto/internals/v1/reminders.proto

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+
+package dapr.proto.internals.v1;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/dapr/dapr/pkg/proto/internals/v1;internals";
+
+// Reminder represents a reminder that is stored in the Dapr actor state store.
+message Reminder {
+  string actor_id = 1;
+  string actor_type = 2;
+  string name = 3;
+  bytes data = 4;
+  string period = 5;
+  google.protobuf.Timestamp registered_time = 6;
+  string due_time = 7;
```

Comment:
Why `string` and not `google.protobuf.Timestamp` for `due_time`?

Reply:
it's the value as sent by the user, and it could be a duration such as "2m". A bit of a legacy thing.